Tasks/LambdaInvokeFunction/TaskOperations.ts

```diff
@@ -49,6 +53,17 @@ export class TaskOperations {
         }
     }
 
+    // A quick heuristic for if it is json. In the past, it was possible to put in just a string
+    // which is not valid JSON. This heurstic checks for if it is a JSON string, object, or array
+    private isJson(input: string): boolean {
```

Comment:
Nit: Based on the `helpMarkDown`, you may want to change the function name to something like `isValidPayload`

Reply:
But we're checking if it's JSON not if it's a valid payload. The only type of valid payload is JSON according to the JS SDK docs

Reply:
Fair, forgot that a string is JSON.